generator-jhipster-gomicro/generators/server/generator.mjs

```diff
@@ -121,6 +123,28 @@ export default class extends ServerGenerator {
   get [ServerGenerator.WRITING]() {
     return {
       writing() {
+        const matchingScenarios= findConfigByBaseName(this.baseName)
+        console.log(matchingScenarios)
+        
+        if (matchingScenarios.length > 0) {
+          var restServer, restClient, rabbitmqServer, rabbitmqClient;
+        
+          for (var options of matchingScenarios) {
+            if (options.framework === 'rest-api') {
+              if (options.server)
+                restServer = options.server;
+              if (options.client)
+                restClient = options.client;
+            } else if (options.framework === 'rabbitmq') {
+              if (options.server)
+                rabbitmqServer = options.server;
+              if (options.client)
+                rabbitmqClient = options.client;
+            }
+          }
+        }
+        
+        console.log("%%%%%%%%%%%%%%",restServer,restClient)
```

Comment:
Please remove this log or have some meaning full log,
having a meaning full log would be a good option